route/person.js

```diff
@@ -31,30 +32,30 @@ function createPerson(req, res, next) {
   });
 }
 
-function queryPeople(req, res, next) {
-  req.logger.info('Querying people', req.query);
+function queryPerson(req, res, next) {
+  req.logger.info('Querying person', req.query);
   req.model('Person').countAndFind(req.query)
     .skip(req.skip)
     .limit(req.limit)
     .sort(req.sort)
     .lean()
-    .exec((err, people, peopleCount) => {
+    .exec((err, person, personCount) => {
       if (err) { return next(err); }
 
-      req.logger.verbose('Sending people to client');
-      res.sendQueried(people, peopleCount);
+      req.logger.verbose('Sending person to client');
+      res.sendQueried(person, personCount);
```

Comment:
I normalize several cases where people was used instead of person (without relationship with plural). Anyway I think is better to you `persons` for plural to avoid confusion. I'll change that